static/app/views/performance/cache/samplePanel/charts/useTransactionDurationSeries.ts

```diff
@@ -0,0 +1,59 @@
+import keyBy from 'lodash/keyBy';
+
+import type {Series} from 'sentry/types/echarts';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {getSeriesEventView} from 'sentry/views/starfish/queries/getSeriesEventView';
+import type {MetricTimeseriesRow} from 'sentry/views/starfish/queries/useSpanMetricsSeries';
+import {useWrappedDiscoverTimeseriesQuery} from 'sentry/views/starfish/utils/useSpansQuery';
+
+// TODO - this is pretty much the same as `useSpanMetricsSeries`, we should probably consider making useMetricsSeries later on
```

Comment:
Ack, yeah! The merge should be very easy! The hook has specs and everything, you'd just need to rename it, accept the `dataset` prop, and move it to `performance` folder

Reply:
honestly, I wanted to, but I try not to mix feature additions with refactors too much.

Reply:
I always prefer to do refactors separately and first! Just 'cause it's so much easier to review